pages/agent/plugins.md.erb

```diff
@@ -0,0 +1,166 @@
+
+# Buildkite Agent Plugins
+
+__Plugins are only available in the beta 3.0 version of the agent__
+
+Plugins provide a way to extend the functionality that Buildkite natively offers. 
+Essentially, they are  collections of [Hooks][hooks] that run as part of your 
```

Comment:
There's a few of these "Essentially" that might be good to remove

Reply:
Removed a broken link, could only find this one `Essentially`, but I removed it.